proto/src/op/message.rs

```diff
@@ -526,7 +531,8 @@ impl Message {
                 records.push(record)
             } else {
                 match record.rr_type() {
-                    RecordType::SIG => {
+                    #[cfg(feature = "dnssec")]
+                    RecordType::DNSSEC(DNSSECRecordType::SIG) => {
```

Comment:
Technically speaking this is SIG0 signing, not DNSSec. i.e. used for dynamic update. Should we create a separate feature for this?